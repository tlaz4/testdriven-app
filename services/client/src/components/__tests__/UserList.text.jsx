import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import UsersList from '../UsersList'

const users = [
	{
		'active': true,
		'email': 'tlaz4@test.com',
		'id': 1,
		'username': 'tlaz'
	},
	{
		'active': true,
		'email': 'tyler@test.com',
		'id': 2,
		'username': 'tyler'
	}
];

test('UsersList renders properly', () => {
	const wrapper = shallow(<UsersList users={users}/>);
	const element = wrapper.find('h4');
	expect(element.length).toBe(2);
	expect(element.get(0).props.children).toBe('tlaz')
});

test('UsersList renders a snapshot properly', () => {
	const tree = renderer.create(<UsersList users={users}/>).toJSON();
	expect(tree).toMatchSnapshot();
});