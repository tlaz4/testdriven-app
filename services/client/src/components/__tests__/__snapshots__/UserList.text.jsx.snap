// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList renders a snapshot properly 1`] = `
<div>
  <h4
    className="box title is-4"
  >
    tlaz
  </h4>
  <h4
    className="box title is-4"
  >
    tyler
  </h4>
</div>
`;
